### ISSUE TITLE
Not working in arc

### ISSUE THREAD

**Issue by Visual-Studio-Coder** [nchudleigh/homerow#133]
**Describe the bug**

I did the hotkey, and no tags show up on web elements in arc browser. i updated to sequoia this morning btw

**Screenshots / video**

Add screenshots to help explain your problem. You can also use Loom to easily record your screen to show the bug in action. That is *really* helpful!

**Steps to reproduce the bug**

**Comment by flavour-of-qualia** [comment]
Yes, for me it does not work as well. Brave also doesot working. Maybe the problem is with chromium itself?

**Comment by devnoname120** [comment]
Weird it has always worked for me with the option `Browser labels` set to `Exhaustive`:

<img width="1291" alt="image" src="https://github.com/user-attachments/assets/7dc56679-d953-4bd3-9c32-5cb962f0b382" />

**My config:**
- macOS Sequoia 15.2 (24C101)
- Arc 1.74.0 (57065)
- Homerow 1.3.0

**Comment by jgraber-io** [comment]
I've been trying to nail down some useful feedback on this and I think I finally have some.  My screenshots are from Google News, but I also had the same results on other sites (like GitHub, for example).

If a page and has no labels for links:

![Image](https://github.com/user-attachments/assets/64f7bd37-e176-485c-b4c5-1eac415086ff)

If I click away to another app (Finder, for example), click back on Arc, and hit my Homerow shortcut again, the labels appear:

![Image](https://github.com/user-attachments/assets/d758dad2-7b01-4a18-87ff-54a9abb56304)

Hope this helps!

**Comment by godbout** [comment]
@jgraber-io https://github.com/nchudleigh/homerow/issues/124#issuecomment-2308892317

**Comment by jgraber-io** [comment]
Thanks, @godbout 🙏 

With the lack of Arc development from The Browser Company these days, I'm not optimistic they will fix the underlying problem.

Fwiw, this happens while actively using the browser, too.  I can be in Gmail with Homerow working fine, for example, and the next time I call it the email pane will have no links 🤷‍♂

**Comment by Visual-Studio-Coder** [comment]
The problem of labels not showing up in Arc can be solved by zapping certain parts of your page (but it depends on the site)

I think its a temporary and inconvenient solution though
